### DISCUSSION TITLE
[Database] 博客不显示Notion中引用的Database

### DISCUSSION THREAD

**Discussion by KashiwaByte** [tangly1024/NotionNext#1970] [Q&A]
**描述bug**
在Notion页面中引用database的view，但是在网站中无法显示

**复现步骤**
任意一篇文章，点击加载database的view即可
![image](https://github.com/tangly1024/NotionNext/assets/106975133/eb3a3645-ae61-4d32-b1a3-70a41d1c3800)

**期望的正常结果**
希望能在网页中显示database的view

**截图**
相关的页面，应该的结果
![image](https://github.com/tangly1024/NotionNext/assets/106975133/85f0ffcb-0658-4620-85b9-47e6f88bdb94)
![image](https://github.com/tangly1024/NotionNext/assets/106975133/47e7bebf-83b6-47a7-a126-5acf094a8b1d)

**环境**

- 操作系统: windows
- 浏览器 edge
- NotionNext版本 4.05
- 主题 全部主题

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
似乎此问题已经解决

Database转为 Inline即可
